soc/arm/nordic_nrf/nrf52/dts_fixup.h

```diff
@@ -2,166 +2,30 @@
-#define DT_QDEC_A_PIN		DT_NORDIC_NRF_QDEC_40012000_A_PIN
-#define DT_QDEC_B_PIN		DT_NORDIC_NRF_QDEC_40012000_B_PIN
-#if defined(DT_NORDIC_NRF_QDEC_40012000_LED_PIN)
-#define DT_QDEC_LED_PIN		DT_NORDIC_NRF_QDEC_40012000_LED_PIN
-#endif
-#if defined(DT_NORDIC_NRF_QDEC_40012000_ENABLE_PIN)
-#define DT_QDEC_ENABLE_PIN		DT_NORDIC_NRF_QDEC_40012000_ENABLE_PIN
+#define DT_GPIO_P1_DEV_NAME		DT_NORDIC_NRF_GPIO_GPIO_1_LABEL
 #endif
-#define DT_QDEC_LED_PRE		DT_NORDIC_NRF_QDEC_40012000_LED_PRE
-#define DT_QDEC_STEPS		DT_NORDIC_NRF_QDEC_40012000_STEPS
-
-#define DT_SPI_0_BASE_ADDRESS       DT_NORDIC_NRF_SPI_40003000_BASE_ADDRESS
-#define CONFIG_SPI_0_NAME               DT_NORDIC_NRF_SPI_40003000_LABEL
-#define CONFIG_SPI_0_IRQ_PRI            DT_NORDIC_NRF_SPI_40003000_IRQ_0_PRIORITY
-#define DT_SPI_0_IRQ                DT_NORDIC_NRF_SPI_40003000_IRQ_0
-#define DT_SPI_0_NRF_SCK_PIN	DT_NORDIC_NRF_SPI_40003000_SCK_PIN
-#define DT_SPI_0_NRF_MOSI_PIN	DT_NORDIC_NRF_SPI_40003000_MOSI_PIN
-#define DT_SPI_0_NRF_MISO_PIN	DT_NORDIC_NRF_SPI_40003000_MISO_PIN
-#define DT_SPI_0_NRF_CSN_PIN	DT_NORDIC_NRF_SPI_40003000_CSN_PIN
-
-#define DT_SPI_1_BASE_ADDRESS       DT_NORDIC_NRF_SPI_40004000_BASE_ADDRESS
-#define CONFIG_SPI_1_NAME               DT_NORDIC_NRF_SPI_40004000_LABEL
-#define CONFIG_SPI_1_IRQ_PRI            DT_NORDIC_NRF_SPI_40004000_IRQ_0_PRIORITY
-#define DT_SPI_1_IRQ                DT_NORDIC_NRF_SPI_40004000_IRQ_0
-#define DT_SPI_1_NRF_SCK_PIN	DT_NORDIC_NRF_SPI_40004000_SCK_PIN
-#define DT_SPI_1_NRF_MOSI_PIN	DT_NORDIC_NRF_SPI_40004000_MOSI_PIN
-#define DT_SPI_1_NRF_MISO_PIN	DT_NORDIC_NRF_SPI_40004000_MISO_PIN
-#define DT_SPI_1_NRF_CSN_PIN	DT_NORDIC_NRF_SPI_40004000_CSN_PIN
 
-#define DT_SPI_2_BASE_ADDRESS       DT_NORDIC_NRF_SPI_40023000_BASE_ADDRESS
-#define CONFIG_SPI_2_NAME               DT_NORDIC_NRF_SPI_40023000_LABEL
-#define CONFIG_SPI_2_IRQ_PRI            DT_NORDIC_NRF_SPI_40023000_IRQ_0_PRIORITY
-#define DT_SPI_2_IRQ                DT_NORDIC_NRF_SPI_40023000_IRQ_0
-#define DT_SPI_2_NRF_SCK_PIN	DT_NORDIC_NRF_SPI_40023000_SCK_PIN
-#define DT_SPI_2_NRF_MOSI_PIN	DT_NORDIC_NRF_SPI_40023000_MOSI_PIN
-#define DT_SPI_2_NRF_MISO_PIN	DT_NORDIC_NRF_SPI_40023000_MISO_PIN
-#define DT_SPI_2_NRF_CSN_PIN	DT_NORDIC_NRF_SPI_40023000_CSN_PIN
+#define CONFIG_I2C_0_NAME		DT_NORDIC_NRF_I2C_I2C_0_LABEL
+#define CONFIG_I2C_1_NAME		DT_NORDIC_NRF_I2C_I2C_1_LABEL
 
-#define DT_SPI_3_BASE_ADDRESS       DT_NORDIC_NRF_SPI_4002B000_BASE_ADDRESS
-#define CONFIG_SPI_3_NAME               DT_NORDIC_NRF_SPI_4002B000_LABEL
-#define CONFIG_SPI_3_IRQ_PRI            DT_NORDIC_NRF_SPI_4002B000_IRQ_0_PRIORITY
-#define DT_SPI_3_IRQ                DT_NORDIC_NRF_SPI_4002B000_IRQ_0
-#define DT_SPI_3_NRF_SCK_PIN	DT_NORDIC_NRF_SPI_4002B000_SCK_PIN
-#define DT_SPI_3_NRF_MOSI_PIN	DT_NORDIC_NRF_SPI_4002B000_MOSI_PIN
-#define DT_SPI_3_NRF_MISO_PIN	DT_NORDIC_NRF_SPI_4002B000_MISO_PIN
-#define DT_SPI_3_NRF_CSN_PIN	DT_NORDIC_NRF_SPI_4002B000_CSN_PIN
+#define DT_QDEC_NAME			DT_NORDIC_NRF_QDEC_QDEC_0_LABEL
```

Comment:
Do we need this? It does not seem to be used anywhere (apart from the driver that is updated in this PR).
Same story with `DT_USBD_NRF_NAME`.